cosmwasm/deploy-contract.js

```diff
@@ -162,8 +167,16 @@ const makeGatewayInstantiateMsg = ({ Router: { address: routerAddress }, VotingV
     return { router_address: routerAddress, verifier_address: verifierAddress };
 };
 
-const makeMultisigProverInstantiateMsg = (contractConfig, contracts, { id: chainId }) => {
+const makeMultisigProverInstantiateMsg = (config, chainName) => {
+    const {
+        axelar: { contracts, chainId: network, axelarId },
```

Comment:
```suggestion
        axelar: { contracts, chainId: axelarChainId, axelarId },
```
maybe or `axelarNetworkId`?